docs/overview.rst

```diff
@@ -300,15 +300,28 @@ would translate into the following declarative model:
     >>> log.id
     ...
 
+.. _auto-generated-identifiers:
 
-Auto-generated primary key
+Auto-generated identifiers
 ..........................
 
+CrateDB does not provide traditional sequences or ``SERIAL`` data type support,
+which enable automatically assigning incremental values when inserting records.
+However, it offers server-side support by providing an SQL function to generate
+random identifiers of ``STRING`` type, and client-side support for generating
+``INTEGER``-based identifiers on behalf of the SQLAlchemy dialect.
```

Comment:
Not sure what "on behalf of" means here - does this mean "when using the SQLAlchemy dialect" or something else?  "on behalf of" implies there's some ID generating function that can be plugged into SQLAlchemy.  I don't know SQLAlchemy so am not sure what happens here.

Reply:
"when using the SQLAlchemy dialect" is better. Thanks.